src/product/productSuggestions.ts

```diff
@@ -3,8 +3,9 @@ import { Product } from '../types/ProductPageData'
 /**
  * @TODO: implement
  */
-function productSuggestions(params, req, res): Promise<Product[]> {
-  return Promise.resolve([]);
+async function productSuggestions(params, req, res): Promise<Product[]> {
+  const result = await Promise.resolve([]);
+  return result;
```

Comment:
I don't see why this is needed. This should be equivalent to the previous implementation. I suspect your changes to the index file was the real fix.